lmfit/minimizer.py

```diff
@@ -1497,34 +1493,38 @@ def emcee(self, params=None, steps=1000, nwalkers=100, burn=0, thin=1,
                                       handle_inf=False)
 
         # If uncertainty was automatically estimated, weight the residual properly
-        if (not is_weighted) and (result.residual.size > 1):
-            if '__lnsigma' in params:
-                result.residual = result.residual/np.exp(params['__lnsigma'].value)
+        if (
+            (not is_weighted)
+            and (result.residual.size > 1)
+            and '__lnsigma' in params
+        ):
```

Comment:
I am okay with folding the second `if`-statement in here, but don't like the formatting - I don't think all these parenthesis are actually needed or (I know the were there already...).
Can this not just become:
```
if not is_weighted and result.residual.size > 1 and '__lnsigma' in params:
```